Argo/Functions/catDecoded.swift

```diff
@@ -1,9 +1,32 @@
+/**
+  Remove `Failure` values from an array and unwrap `Success`es
+
+  This will iterate through the array of `Decoded` values and ensure that each
+  is `Success`. If it is, it will unwrap the contained `T` value. If it is
+  `Failure`, it will be removed from the array.
+
+  - parameter xs: An array of `Decoded` values
```

Comment:
I think using the full type signature is ok ... `Decoded<T>` ... it also makes more sense as you read on